pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/Flake8Task.java

```diff
@@ -32,8 +34,37 @@
 public class Flake8Task extends AbstractPythonMainSourceDefaultTask {
 
     private static final Logger log = Logging.getLogger(Flake8Task.class);
+    // Track whether the current run is excluding the new rules
+    private Boolean ignoringNewRules = false;
+    private String firstRunOutput = null;
+
+    /* Set of Flake8 rules that were added in version bumps. This set of rules, and this comment should be updated
+       every time the flake8 version is bumped.
+
+       Current flake8 version:
+       Previous flake8 version: 3.6.0
+       Rules added between versions:
+     */
+    private static final Set<String> IGNORE_RULES = new HashSet<>(Arrays.asList(
+        // Empty for now until we bump to flake8 3.8.0
+    ));
+
+    private Set<String> ignoreRules = IGNORE_RULES;
+
+    private static final String IGNORED_RULES_MSG = "######################### WARNING ##########################\n"
+        + "The flake8 version has recently been bumped, which add the following new rules:\n"
+        + "%s\n"  // This will be replaced with the set of ignored rules
+        + "Your project is failing for one or more of these rules. Please address them, as they will be enforced soon.\n"
+        + "%s############################################################\n";
+
+    // Provide the ability to override the ignored rules for testing purposed, but mainly so the LI side can maintain
```

Comment:
s/purposed/purposes/

others may need to do similar things besides us, so I'd change the rest of the comment to: "so that users can enforce the use of a different version of flake8 in their plugins."